safe_transaction_service/history/views.py

```diff
@@ -180,7 +180,8 @@ class AllTransactionsListView(ListAPIView):
         django_filters.rest_framework.DjangoFilterBackend,
         OrderingFilter,
     )
-    ordering_fields = ["execution_date", "safe_nonce", "block", "created"]
+    ordering_fields = ["execution_date"]
+    allowed_ordering_fields = ["execution_date", "-execution_date"]
```

Comment:
```suggestion
    allowed_ordering_fields = ordering_fields +  [f'-{ordering_field}' for ordering_field in ordering_fields]
```
This should be programatic to prevent typos